flink-runtime/src/main/java/org/apache/flink/runtime/metrics/dump/MetricDumpSerialization.java

```diff
@@ -50,12 +51,27 @@
 	private MetricDumpSerialization() {
 	}
 
+	public static class MetricSerializationResult {
```

Comment:
This is not `Serializable` and would fail when send with Akka. The following test fails:
```java 
@Test
public void testJavaSerialization() throws IOException {
	MetricDumpSerialization.MetricDumpSerializer serializer = new MetricDumpSerialization.MetricDumpSerializer();

	final ByteArrayOutputStream bos = new ByteArrayOutputStream(1024);
	final ObjectOutputStream oos = new ObjectOutputStream(bos);

	oos.writeObject(serializer.serialize(
		new HashMap<Counter, Tuple2<QueryScopeInfo,String>>(),
		new HashMap<Gauge<?>, Tuple2<QueryScopeInfo,String>>(),
		new HashMap<Histogram, Tuple2<QueryScopeInfo,String>>(),
		new HashMap<Meter, Tuple2<QueryScopeInfo,String>>()));
}
```